pkg/server/server.go

```diff
@@ -31,10 +31,30 @@ type kubeconfigFunc func() (kubeconfigData []byte, rootCAData []byte, err error)
 // appenderFunc appends Config.
 type appenderFunc func(*ignv2_2types.Config) error
 
+// Error is returned by the GetConfig API
+type Error struct {
```

Comment:
we might want to call it `GetConfigError` or something like that (doesn't need uppercase to be exported also, does it?)

Reply:
(@runcom I thought upper to be an exported type)

Reply:
like this naming suggestion btw.

Reply:
yeah, I realized only after commenting that I uppercased my suggestion 😄

Reply:
Or possibly `configError` to be explicit.